pkg/debug/scripts/collect-info.sh

```diff
@@ -43,11 +43,11 @@ usage()
     echo ""
     echo "Collect-logs mode:"
     echo "       -s tar whole /sysfs"
+    echo "       -t NUMBER-OF-DAYS    - read logs from the last NUMBER-OF-DAYS [0-30]. Set to 0 to not include any logs."
```

Comment:
Regarding the help message:
1. `read logs` -> `collect logs`
2. `Set to 0 to not include any logs` ->  `Set to 0 to not include /persist/newlog logs`

Reply:
That makes sense :ok_hand: